core/controllers/reader.py

```diff
@@ -300,7 +300,127 @@ def get(self, exploration_id):
 
 
 class StorePlaythroughHandler(base.BaseHandler):
-    """Handles a useful playthrough coming in from the frontend to store it."""
+    """Handles a useful playthrough coming in from the frontend to store it. If
+    the playthrough already exists, it is updated in the datastore.
+    """
+
+    def __init__(self, request, response):
+        """Method that initializes member variables for the handler."""
+        super(StorePlaythroughHandler, self).__init__(request, response)
```

Comment:
Since you aren't using/making any changes to the input arguments, pass `*args, **kwargs` instead:

```Python
def __init__(self, *args, **kwargs):
    super(StorePlaythroughHandler, self).__init__(*args, **kwargs)
    # initialize members.
```

This will protect your class from any future changes to `__init__` in `base.BaseHandler`.

Reply:
That makes sense. Done.